### PR TITLE
user-get-charging-keys API call doc added

### PR DESCRIPTION
**Pull request by akhrol**

### CODE REVIEW

**Comment by apheleia** [comment]
17 commits?! 😳 Make it one please 😊

File: docs/calls/user_get_charging_keys.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.. highlight:: js
+
+.. _calls-usergetchargingkeys-docs:
+
+User Get Bills
```
Comment (apheleia) [inline docs/calls/user_get_charging_keys.rst]: Wrong title

File: docs/calls/user_get_charging_keys.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        For example: if the identifier type is username and the identifier is the user's username,
+        then token is used for authentication instead of a password.
+
+Response
+--------
+
+Fields
+~~~~~~
+
+charging-keys
```
Comment (apheleia) [inline docs/calls/user_get_charging_keys.rst]:
```
user
  charging-keys
    uid...
```

Comment (akhrol) [inline docs/calls/user_get_charging_keys.rst]: it is explained in request.rst

**Pull request closed without merging**